src/App.jsx

```diff
@@ -14,57 +16,89 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [stateMessege, setStateMessege] = useState(false);
+  const [goodName, setGood] = useState('Jam');
+  let id = 0;
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      {stateMessege && (
+        <h1 className="title is-flex is-align-items-center ">
+          No goods selected
+        </h1>
+      )}
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+      {!stateMessege && (
+        <h1 className="title is-flex is-align-items-center">
+          {goodName} is selected
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => {
+              setStateMessege(true);
+              setGood('');
+            }}
```

Comment:
```suggestion
            onClick={reset}
```
Consider moving this function from jsx